framework/encode/capture_manager.cpp

```diff
@@ -954,6 +1009,21 @@ void CaptureManager::BuildOptionList(const format::EnabledOptions&        enable
     option_list->push_back({ format::FileOption::kCompressionType, enabled_options.compression_type });
 }
 
+format::ApiFamilyId CaptureManager::GetApiFamily() const
+{
+    const format::ApiFamilyId none_id       = format::ApiFamilyId::ApiFamily_None;
+    format::ApiFamilyId       id            = none_id;
+    const size_t              manager_count = api_capture_managers_.size();
+    if (manager_count == 1)
+    {
+        id = api_capture_managers_.cbegin()->first->GetApiFamily();
+    }
+    else if (manager_count > 1)
+    {
+        id = format::ApiFamilyId::ApiFamily_Multiple;
+    }
+    return id;
+}
 void CaptureManager::WriteDisplayMessageCmd(const char* message)
```

Comment:
These following calls come in through the API-specific capture manager, so we should just add the "apiFamily" as an argument and have overrides in the API-specific capture managers which don't take the "apiFamily" but add it for the common capture manager.